remove-duplicates-from-sorted-array/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var removeDuplicates = function(nums) {
+    let l = 0, r = 0
```

Comment:
Do you really need this 2 buffer variables? Should be possible with only one, to save memory and improve speed of you solution.